Core/src/org/sleuthkit/autopsy/datamodel/AbstractAbstractFileNode.java

```diff
@@ -368,18 +327,16 @@ public String toString() {
             properties.add(new NodeProperty<>(ORIGINAL_NAME.toString(), ORIGINAL_NAME.toString(), NO_DESCR, ""));
         }
 
-        //SCO column prereq info..
-        List<ContentTag> tags = getContentTagsFromDatabase();
-        CorrelationAttributeInstance attribute = getCorrelationAttributeInstance();
-
-        Pair<DataResultViewerTable.Score, String> scoreAndDescription = getScorePropertyAndDescription(tags);
-        properties.add(new NodeProperty<>(SCORE.toString(), SCORE.toString(), scoreAndDescription.getRight(), scoreAndDescription.getLeft()));
-        DataResultViewerTable.HasCommentStatus comment = getCommentProperty(tags, attribute);
-        properties.add(new NodeProperty<>(COMMENT.toString(), COMMENT.toString(), NO_DESCR, comment));
-        if (!UserPreferences.hideCentralRepoCommentsAndOccurrences()) {
-            Pair<Long, String> countAndDescription = getCountPropertyAndDescription(attribute);
-            properties.add(new NodeProperty<>(OCCURRENCES.toString(), OCCURRENCES.toString(), countAndDescription.getRight(), countAndDescription.getLeft()));
-        }
+        // Create place holders for S C O 
+        properties.add(new NodeProperty<>(SCORE.toString(), SCORE.toString(), NO_DESCR, ""));
+        properties.add(new NodeProperty<>(COMMENT.toString(), COMMENT.toString(), NO_DESCR, ""));
```

Comment:
Should we have a description like "Value loading" or something to that extent instead of "no description" as the temporary description for these columns.  I ask this because some of many cells will appear to be blank even after they load for these columns and the description is how we were giving the user a little information about what the cell value displayed means. "no description" doesn't really tell them anything.